parking-spot-submodule/src/main/java/ua/lviv/iot/infrastructure/grpc/KafkaGrpcService.kt

```diff
@@ -1,4 +1,4 @@
-package ua.lviv.iot.parkingServer.kafka
+package ua.lviv.iot.infrastructure.grpc
```

Comment:
in one class there are: Kafka (even if just naming), gRPC, NATS) refactor it please so infrastructure components will not depend on each other

Reply:
✅